### DISCUSSION TITLE
Any Successful Bot Purchases? Please Share Your Success!

### DISCUSSION THREAD

**Discussion by abigdreamer** [TreborNamor/BestbuyBot#52] [Show and tell]
Please share source and tell us about anything successful happening with these Bots. Tell us your setup, your proxy if any, and share the source. Tell us what you were able to buy and if any extra steps (baby sitting) was required. 
Thanks.

**Comment by TanikoShima** [comment]
Unsuccessful with any purchases (even testing purchases) as of 6/22/21.
The Amazon bot hangs at the checkout page and cannot seem to click the "Place your order" button. Log hangs at "Attempting to buy . . "
